src/controllers/StudentController.js

```diff
@@ -26,6 +27,12 @@ module.exports = {
         include: [
           { model: Stream, attributes: ['name'] },
           { model: Group, attributes: ['groupName'] },
+          {
+            model: Payment,
+            attributes: ['id', [literal('"date"'), 'maxPaymentDate'], 'amount'],
```

Comment:
Вот так не будет работать?
```suggestion
            attributes: ['id', ['date', 'maxPaymentDate'], 'amount'],
```